config/services/common/services_prod.yml

```diff
@@ -0,0 +1,97 @@
+parameters:
+    ps_cache_dir: !php/const _PS_CACHE_DIR_
```

Comment:
Prefer `%kernel.cache_dir%` instead

Reply:
I agree, but imho it's not the same path

Reply:
Yes, but we need to use `/var/cache/$ENV/` instead of `/cache/` and more when they are Symfony services :D

Reply:
?

`/cache/` is not supposed to be used anymore.